$NetBSD: patch-configure,v 1.9 2020/03/17 19:04:49 adam Exp $

Do not add default compiler flags.
Leave pkgsrc to handle security features.

--- configure.orig	2019-11-29 19:23:06.000000000 +0000
+++ configure
@@ -3721,8 +3721,8 @@ case $host_os in *\ *) host_os=`echo "$h
 
 
 # Add some default CFLAGS and CXXFLAGS, can be appended to using the environment variables
-CFLAGS="-Wall -g -O2 $CFLAGS"
-CXXFLAGS="-Wall -g -O2 $CXXFLAGS"
+CFLAGS="-Wall $CFLAGS"
+CXXFLAGS="-Wall $CXXFLAGS"
 
 ac_ext=c
 ac_cpp='$CPP $CPPFLAGS'
@@ -7884,7 +7884,7 @@ rm -f core conftest.err conftest.$ac_obj
 fi
 { $as_echo "$as_me:${as_lineno-$LINENO}: result: $gl_cv_warn_cxx__fstack_protector" >&5
 $as_echo "$gl_cv_warn_cxx__fstack_protector" >&6; }
-if test "x$gl_cv_warn_cxx__fstack_protector" = xyes; then :
+if test "x$gl_cv_warn_cxx__fstack_protector" = xDISABLED; then :
 
         CFLAGS="-fstack-protector $CFLAGS"
         CXXFLAGS="-fstack-protector $CXXFLAGS"
@@ -22723,7 +22723,7 @@ fi
 
   if test "x$PG_CONFIG" != "x" -o "x$PGSQL_LIBS" = "x"; then :
 
-        if test "x$PG_CONFIG" == "x"; then :
+        if test "x$PG_CONFIG" = "x"; then :
 
       # Extract the first word of "pg_config", so it can be a program name with args.
 set dummy pg_config; ac_word=$2
@@ -22767,7 +22767,7 @@ fi
 
 
 fi
-    if test "x$PG_CONFIG" == "x"; then :
+    if test "x$PG_CONFIG" = "x"; then :
 
       as_fn_error $? "Can not find pg_config, use --with-pg-config to specify the path to pg_config" "$LINENO" 5
 
